### DISCUSSION TITLE
Client config on Windows 10

### DISCUSSION THREAD

**Discussion by DCA-Team-IT** [rustdesk/rustdesk#1540] [Q&A]
Hi,

I Installed a Server for us which is running fine.
And I installed the client with the silent install switch (only this). When I rename the rustdesk.exe to rustdesk-host=<host-ip-or-name>,key=<public-key-string>.exe with our data I can see it in the about Window but the client wont connect and shows the UAC Message.
The client do connect when I put the data in ID/relayserver option within the client. so this is fine BUT when I copy the config files to another computer the files are getting overwritten with whatever is set on that computer. Also when I login to a new user account this account also can use the server without making any entries.

So my question is: where is this data stored on the computer? I want to install the client on serveral PCs and like to have the id and key prefilled so it works "out of the box" for any user.
I searched the registry and only found something  under uninstall, but changes anything doesn't make a difference. under ProgrammData I also can't find files.

greetings

**Comment by ghost** [comment]
Hi,
i'd recommand to check this issues. Your question has been answered:
https://github.com/rustdesk/rustdesk/issues/838

**Reply by DCA-Team-IT** [reply]
thanks sorry I overlooked that
